Src/IronPython/Runtime/Bytes.cs

```diff
@@ -52,7 +51,11 @@ public class Bytes : IList<byte>, IReadOnlyList<byte>, IEquatable<Bytes>, ICodeF
         [StaticExtensionMethod]
         public static object __new__(CodeContext context, [NotNull] PythonType cls, [NotNull] IBufferProtocol source) {
             if (cls == TypeCache.Bytes) {
-                return new Bytes(source);
+                if (TryInvokeBytesOperator(context, source, out Bytes? res)) {
```

Comment:
How about a separate overload?

Reply:
What sort of overload?

Reply:
In 05ceb50 I added an overload for the C# constructor with a `Bytes` parameter as a fast track. I was also thinking about an overload for the Python `__new__` constructor, but this would not work: it would catch also the subclasses and those have to be tried with `__bytes__`.